rita/src/rita_common/port_emissary/mod.rs

```diff
@@ -0,0 +1,35 @@
+extern crate netstat;
+
+use rita_common::port_emissary::netstat::*;
+
+struct PortEmissary {
+    free_ports: Vec<u16>,
+}
+
+impl PortEmissary {
+    fn new() -> PortEmissary {
+        let af_flags = AddressFamilyFlags::IPV4 | AddressFamilyFlags::IPV6;
+        let sockets_info = get_sockets_info(af_flags, ProtocolFlags::UDP).unwrap();
```

Comment:
if we're running this more than once at startup this unwrap needs to be handled.